components/authz-service/config/config_mgmt_test.go

```diff
@@ -0,0 +1,155 @@
+package config_test
+
+import (
+	"io/ioutil"
+	"os"
+	"testing"
+
+	"github.com/chef/automate/components/authz-service/config"
+	event_ids "github.com/chef/automate/lib/event"
+	"github.com/stretchr/testify/assert"
+)
+
+const cFile = "/tmp/.authz-service.toml"
```

Comment:
If the name of this file doesn't matter to the app code, something that mentioned this was a test file would be nice so it doesn't get mistaken for something important and left around.

Reply:
I renamed the file like this `/tmp/.authz-service-test-delete-me.toml`